### ISSUE TITLE
IoU metric implementation

### ISSUE THREAD

**Issue by fmagera** [vcg-uvic/sportsfield_release#12]
Hi, 
I don't understand the code for the evaluation on the world cup dataset. There seems to be a discretization of the soccer pitch shape : https://github.com/vcg-uvic/sportsfield_release/blob/master/utils/metrics.py#L94
and thresholding then :  
https://github.com/vcg-uvic/sportsfield_release/blob/master/utils/metrics.py#L102
Is it an approximation of the IoU or is there something that I am missing ? Thanks for your help :)

**Comment by jiangwei221** [comment]
Yes, this is kind of a approximation if IoU using rasterization(simple resample).
We didn't go with the geometric way, for example using Shapely.
